### PR TITLE
Feat: bookmark, comments, like

### PR DESCRIPTION
**Pull request by yashpatel1797**
feature
- Bookmark
- like
- comments

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work
code looks clean.
keep it up.

File: src/components/Comments/Comments.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { Link } from 'react-router-dom';
+
+const Comments = ({ comment,
+    username,
+    firstName,
+    lastName,
+    commentDate,
+    profileImage, }) => {
+    console.log(comment);
+    console.log(firstName);
```
Comment (sharmayatendra) [inline src/components/Comments/Comments.js]: remove console

File: src/pages/SinglePost/SinglePost.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    const { firstName } = user;
+    const { posts } = usePost();
+    const navigate = useNavigate();
+    const [comment, setComment] = useState("");
+    const post = posts.find(post => post._id === postId)
+    console.log(post);
+    const profileImage = false
+    const commentSubmitHandler = (e) => {
+        e.preventDefault();
+        console.log(postId, comment);
```
Comment (sharmayatendra) [inline src/pages/SinglePost/SinglePost.js]: here also

File: src/pages/SinglePost/SinglePost.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const SinglePost = () => {
+    const { postId } = useParams();
+    const dispatch = useDispatch();
+    const { user } = useAuth();
+    const { firstName } = user;
+    const { posts } = usePost();
+    const navigate = useNavigate();
+    const [comment, setComment] = useState("");
+    const post = posts.find(post => post._id === postId)
+    console.log(post);
```
Comment (sharmayatendra) [inline src/pages/SinglePost/SinglePost.js]: here also remove console

**Review by sharmayatendra — APPROVED** [review]
good work
look upon suggested changes

**Pull request merged**